packages/ai/src/prompts/__tests__/chat.test.ts

```diff
@@ -0,0 +1,316 @@
+import { ContentPart, Message } from '../../message';
+import { IChatModel } from '../../models';
+import { Schema } from '../../schema';
+import { ChatPrompt, ChatPromptPlugin } from '../chat';
+
```

Comment:
to match the other projects, tests should use `*.spec.ts` names and be in the same folder location as the source code. This is just to keep this consistent, im open to a discussion of using a tests folder instead but that would require we update the existing packages with tests to match that.

Reply:
ah shoot. This had to be deleted. I already had a `chat.spec.ts`. removed!